.github/workflows/test.yml

```diff
@@ -16,12 +16,43 @@ jobs:
       - name: Cache build
         uses: actions/cache@v4
         with:
-          path: backend/target
-          key: cargo-test-${{ hashFiles('backend/Cargo.lock') }}
+          path: target
+          key: cargo-test-${{ hashFiles('Cargo.lock') }}
```

Comment:
Whoops - I forgot to update this after converting to a workspace. I think that means our CI cache was not getting pruned. Not a huge deal.